src/geo-objects/ObjectManager.js

```diff
@@ -56,6 +57,17 @@ class ObjectManager extends React.Component {
       instance.objects.options.set(objects || {})
     }
 
+    if (!isEqual(features, prevFeatures)) {
+      const prevIds = prevFeatures.map(feature => feature.id)
+      const currentIds = features.map(feature => feature.id)
+      const toBeRemovedIds = difference(prevIds, currentIds)
+      const toBeAddedIds = difference(currentIds, prevIds)
+      const toBeAddedObjects = features.filter(feature => toBeAddedIds.includes(feature.id))
+
+      instance.remove(toBeRemovedIds)
+      instance.add(toBeAddedObjects)
```

Comment:
Okay, that is a valid point! Please move lodash.difference to dev deps and I will merge this